src/Utopia/Messaging/Adapter.php

```diff
@@ -138,6 +152,21 @@ protected function requestMulti(
             throw new \Exception('No URLs provided. Must provide at least one URL.');
         }
 
+        foreach ($headers as $header) {
+            if (\str_contains($header, 'application/json')) {
```

Comment:
I think we use a switch case in other places, might be nicer.

Reply:
We can't switch (nicely) here since we only do a partial match